app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt

```diff
@@ -34,12 +35,25 @@ class LoginFragment : WolmoFragment<FragmentLoginBinding, LoginPresenter>(), Log
         }
     }
 
-    override fun logInUser() = HomeActivity.start(requireContext())
+    override fun logInUser() {
+        binding.indeterminateBar.visibility = View.GONE
```

Comment:
You can make a general function like `toggleProgressBarPresence(isPresent: Boolean)`

Reply:
Noted, will change it in the next PR.